pkg/users/delete-group-dialog.js

```diff
@@ -51,12 +51,16 @@ export function delete_group_dialog(group) {
                 caption: group.userlistPrimary.length > 0 ? _("Force delete") : _("Delete"),
                 style: "danger",
                 clicked: () => {
-                    const prog = ["/usr/sbin/groupdel"];
+                    const prog = ["groupdel"];
```

Comment:
Please do another separate PR (or at least commit) for dropping all the hardcoded paths.

Reply:
Still relevant, but I'll let that slide. Let's do this in a separate PR.

Reply:
Oops sorry missed this one.